src/bytes/spec/bytes_spec.gobra

```diff
@@ -0,0 +1,41 @@
+package spec
+
+// +gobra
+
+import (
+	sl "gobra-libs/byteslice"
+)
+
+ghost
+requires acc(sl.Bytes(ub, 0, len(ub)), _)
+ensures len(res) == len(ub)
+ensures forall i int :: { res[i] } 0 <= i && i < len(ub) ==>
+    res[i] == sl.Byte(ub, 0, len(ub), i)
+decreases _
+pure func View(ub []byte) (res seq[byte])
+
+// TODO: make this opaque
+ghost
+requires count >= 0
+decreases count
+pure func SpecRepeat(b seq[byte], count int) (res seq[byte]) {
+	return count == 0 ? seq[byte]{} : ( b ++ SpecRepeat(b, count - 1) )
+}
+
+
+ghost
+requires low <= high
+ensures res == exists i int :: {i in s} i in s && low <= i && i < high
+opaque
+decreases high - low
+pure func SetContainsInRange(s set[int], low, high int) (res bool) {
+	return low < high && (low in s || ( low < high && SetContainsInRange(s, low + 1, high) ))
```

Comment:
I would move this to some other package, it is not clear why it is defined here

Reply:
I put this function there because it is fairly specific to `Count` but also appears in the postcondition and hence cannot be private

Reply:
My point is that this could be in a aux.gobra file (or sth like that) in the `bytes` pkg, I think it might be confusing that `bytes/spec` contains auxiliary definitions, but no actual specification of the operations in `bytes`